scripts/gen_dockerfile_test.py

```diff
@@ -179,12 +181,14 @@ def test_generate_dockerfile_command(tmpdir, testdata_dir):
     compare_against_golden_files(app, config_dir, testdata_dir)
 
 
+@pytest.mark.parametrize('app', [
+    # Sampled from https://github.com/GoogleCloudPlatform/python-docs-samples
+    'hello_world',
+    'hello_world_compat'])
```

Comment:
The hello_world_compat code didn't come from the python-docs-samples repo, that I can see.  Did you just write something from scratch, or was it from some other sample?  Either way, can add a comment for hello_world_compat, ideally with a URL?